study.md

```diff
@@ -21,18 +21,18 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+Still confused why some of the examples have "var" in them. And also I think i need a refresher on what forEach does again.
 ```
```

Comment:
That may just be our fault for not catching them in our material. Regarding outside material, just think of `let = var` and `var = let`. They are the same things for right now.